sdk/batch/batch/CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Release History
 
+## 11.1.0 (2024-02-01)
```

Comment:
Pretty sure this should be the release date of the SDK, not the API version

Reply:
Updated with 2024-04-03